kedro-datasets/kedro_datasets/geopandas/geojson_dataset.py

```diff
@@ -50,12 +58,13 @@ class GeoJSONDataset(
     """
 
     DEFAULT_LOAD_ARGS: dict[str, Any] = {}
-    DEFAULT_SAVE_ARGS = {"driver": "GeoJSON"}
+    DEFAULT_SAVE_ARGS: dict[str, Any] = {}
 
     def __init__(  # noqa: PLR0913
         self,
         *,
         filepath: str,
+        file_format: str = "file",
```

Comment:
I think it's fine to set a default value, especially because this ensures backward compatibility

Reply:
We don't need this anymore because of backward compatibility, but it's more or less the generic method of geopandas, and it saves some boilerplate in most cases.